### DISCUSSION TITLE
Welcome to admb Discussions!

### DISCUSSION THREAD

**Discussion by johnoel** [admb-project/admb#213] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mclawsonSPR** [comment]
My name is Mike Clawson, I got my MS and PHD at the University of Washington. I live in western Washington.

I use ADMB to run Statistical Population Reconstruction (SPR) models. SPR models are like fisheries stock assessment models for terrestrial species. One of the main features of ADMB I use is the ability to use random effects in a maximum likelihood framework.

Working with the folks at the Columbia Basin Reseach lab at UW we also developed a GUI program that creates and runs SPR models in ADMB based on user input. 

Excited to hear from others about what they use ADMB for. 

Thanks,
Mike
